### PR TITLE
Features: Add CustomCommandKeybindsFeature

### PR DESCRIPTION
**Pull request by kristofbolyai**
Fun fact: This allows users to use keybinds to set/reset configs. Full screen map done. :P

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/user/CustomCommandKeybindsFeature.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    private String keybindCommand3 = "";
+
+    @Config
+    private String keybindCommand4 = "";
+
+    @Config
+    private String keybindCommand5 = "";
+
+    @Config
+    private String keybindCommand6 = "";
```
Comment (Incompleteusern) [inline common/src/main/java/com/wynntils/features/user/CustomCommandKeybindsFeature.java:33]: Code dup is annoying, but necessary as of now

**Comment by Incompleteusern** [comment]
> Fun fact: This allows users to use keybinds to set/reset configs. Full screen map done. :P

Not quite, as only one command can be run per keybind

File: common/src/main/java/com/wynntils/features/user/CustomCommandKeybindsFeature.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    @RegisterKeyBind
+    private final KeyBind executeKeybind6 = new KeyBind(
+            "Execute 6th Custom Command Keybind",
+            GLFW.GLFW_KEY_UNKNOWN,
+            true,
+            () -> this.executeKeybind(keybindCommand6));
+
+    private void executeKeybind(String keybindCommand) {
+        keybindCommand = keybindCommand.trim();
```
Comment (Incompleteusern) [inline common/src/main/java/com/wynntils/features/user/CustomCommandKeybindsFeature.java]: Modifcations should be done only once in `onConfigUpdate`

File: common/src/main/java/com/wynntils/core/features/FeatureRegistry.java [hunk 3]
```diff
@@ -67,6 +68,72 @@
 public final class FeatureRegistry {
     private static final List<Feature> FEATURES = new ArrayList<>();
 
+    public static void init() {
```
Comment (magicus) [inline common/src/main/java/com/wynntils/core/features/FeatureRegistry.java:71]: You moved it to make it easier to find?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/features/FeatureRegistry.java:71]: Yes, this is literally the only method we touch in the class.

**Review by magicus — COMMENTED** [review]
Would be nice if we could get to a point where we could generalize this. I agree with Incomplete's idea on cleaning up the command at config time. Otherwise it looks good.

**Review by Incompleteusern — APPROVED** [review]

**Pull request merged**